components/validateur-report/index.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types'
+import {Pane} from 'evergreen-ui'
+import {groupBy, remove} from 'lodash'
+
+import VoieRow from '@/components/validateur-report/voie-row'
+
+const validAlertSchemas = new Set(['voie_nom', 'numero', 'suffixe'])
+
+function ValidateurReport({rows, voies, baseLocaleId}) {
+  const sanitizedRowsByVoies = {}
+  if (voies) {
+    const rowsByVoies = groupBy(rows, row => row.rawValues.voie_nom)
+
+    // Create a new object, grouped by voies names
+    Object.keys(rowsByVoies).forEach(nomVoie => {
+      const rowsWithAlerts = rowsByVoies[nomVoie].filter(row => {
+        remove(row.errors, error => !validAlertSchemas.has(error.schemaName)) // Remove unusefull alerts
```

Comment:
Il est préférable de créer un objet que d'en muter. Un `pick` ou `filter` ferait très bien l'affaire.

Reply:
`filter` renvoie un nouvel `array`. Or, j'ai seulement besoin de "nettoyer" le tableau `errors` des alertes dont on ne veut pas.

D'après la [documentation](filter(row.errors, error => validAlertSchemas.has(error.schemaName)).length > 0) de Lodash, pour obtenir cet effet, `remove` semble plus adapté car il ne renvoie pas de nouvel `array`

Reply:
Ça implique de le muter, ce qui n'est pas forcément une bonne pratique de manière générale, car on pourrait altérer une donner.

Reply:
Corrigé via le commit [Filter usefull alerts instead of removing](https://github.com/BaseAdresseNationale/mes-adresses/pull/662/commits/f57101ee81b72a2a9bea72be047f09aa3c7c8770)